RNTester/js/TimerExample.js

```diff
@@ -117,28 +116,31 @@ class RequestIdleCallbackTester extends React.Component<{}, $FlowFixMeState> {
   };
 }
 
-var TimerTester = createReactClass({
-  displayName: 'TimerTester',
+type TimerTesterProps = $ReadOnly<{|
+  dt?: number,
+  type: string,
+|}>;
 
-  _ii: 0,
-  _iters: 0,
-  _start: 0,
-  _timerId: (null: ?TimeoutID),
-  _rafId: (null: ?AnimationFrameID),
-  _intervalId: (null: ?IntervalID),
-  _immediateId: (null: ?Object),
-  _timerFn: (null: ?() => any),
+class TimerTester extends React.Component<TimerTesterProps> {
+  _ii = 0;
+  _iters = 0;
+  _start = 0;
+  _timerId = (null: ?TimeoutID);
+  _rafId = (null: ?AnimationFrameID);
+  _intervalId = (null: ?IntervalID);
+  _immediateId = (null: ?Object);
+  _timerFn = (null: ?() => any);
```

Comment:
Flow won't correctly infer the type of these props based on what you're assigning to them. We should instead do:

```javascript
_timeId: ?TimeoutID = null;
_rafId: ?AnimationFrameID = null;
_intervalId: ?IntervalID = null;
_immediateId? Object = null;
_timerFn: ?() => any = null;
```

Reply:
Hmm, I see, I remember seeing the way I have done as a suggestion in one of the PRs. 
Made your suggested changes on this [66a5b27](https://github.com/facebook/react-native/pull/21623/commits/66a5b27bee32b79a0c0bab39e32977de449cab0a)!